torch/distributed/distributed_c10d.py

```diff
@@ -1066,6 +1066,30 @@ def get_backend(group: Optional[ProcessGroup] = None) -> Backend:
     pg_store = _world.pg_map[pg] if pg in _world.pg_map else None
     return Backend(not_none(pg_store)[0])
 
+def get_pg_config() -> List[Dict[str, Union[int, str, List[int]]]]:
+    """
+    Return the pg configuration of all the process groups.
+
+    """
+    config_info: List[Dict[str, Union[int, str, List[int]]]] = []
+    for pg in _world.pg_map.keys():
```

Comment:
The List return and the for loop here make me wonder if this function is actually doing "get-**all**-pg-configs", rather than what its name suggests.

We may want to consider if a "prime" function that's per group:
```
def _get_pg_config(group: Optional(ProcessGroup)=None)
```
would also get the job done.